app/Event/LaravelDispatcherAdapter.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Event;
+
+class LaravelDispatcherAdapter implements Dispatcher
+{
+    /** @var  \Illuminate\Events\Dispatcher */
+    private $dispatcher;
+
+    /**
+     * @param \Illuminate\Events\Dispatcher $dispatcher
+     */
+    public function __construct(\Illuminate\Events\Dispatcher $dispatcher)
+    {
+        $this->dispatcher = $dispatcher;
+    }
+
+    /**
+     * @param object|string $event
+     * @param array         $payload
+     */
+    public function dispatch($event, $payload = [])
+    {
+        $this->dispatcher->dispatch($event, $payload, false);
```

Comment:
**Suggestion:** The `dispatch` method in the `LaravelDispatcherAdapter` class does not return any value. However, the interface `Dispatcher` that this class implements, specifies that the `dispatch` method should return an array or null. To adhere to the contract defined by the interface, you should return the result of the `dispatch` call. [possible issue]
```suggestion
    public function dispatch($event, $payload = [])
    {
        return $this->dispatcher->dispatch($event, $payload, false);
    }
```